### ISSUE TITLE
Communicating with the mappers participating in a job

### ISSUE THREAD

**Issue by severinmenard** [hotosm/osm-tasking-manager#148]
would be really interesting to have the possibility to send messages to the OSM mappers listed into the Users tab

**Comment by maning** [comment]
+1

There are several implementations that might be an inspiration:
- MapCraft - ​http://mapcraft.nanodesu.ru
- JOSM GeoChat plugin - ​http://wiki.openstreetmap.org/wiki/JOSM/Plugins/GeoChat

**Comment by severinmenard** [comment]
Yes I think these tools are great to coordinate in real time. But having the capacity to send an email to the Inbox could be a good way to make the users join these coordinating tools

**Comment by jsanz** [comment]
+1 good idea

**Comment by mikelmaron** [comment]
Duplicate of https://github.com/hotosm/osm-tasking-manager/issues/40
